app/controllers/sessions_controller.rb

```diff
@@ -51,7 +51,11 @@ def do_login
     sign_in(@user) do |status|
       if status.success?
         StatsD.increment "login.success"
-        redirect_back_or(url_after_create)
+        if @user.mfa_enabled?
+          redirect_back_or(url_after_create)
+        else
+          redirect_to new_multifactor_auth_path
```

Comment:
I believe this will redirect for every user that does not have MFA enabled, but initially we will only require MFA for the top 100. How would you feel about adding a "do not show this again" option for those who aren't a part of the top 100? That way we're not being _too_ persistent towards those for whom it's not required

Reply:
While the intent of this prototype was indeed to target all users, you bring up an excellent point Ashley 💯 . This prototype doesn't enforce a consistent narrative, in light of the decision that we've been converging on recently to target only the top 100. So, making adjustments to target the top 100 would make sense if we want to showcase a user flow for a soft rollout targeting that cohort of users only.

I'd love to touch base as a team to verify whether we're scoping down our goal to target top 100, or if we should still explore alternative options that may impact a larger set of users (whether _all users_, or a different subset) 🙌 .